Pod/Classes/Deselection/UIViewController+Deselection.swift

```diff
@@ -30,64 +30,58 @@
 
 import UIKit
 
+public protocol SmoothlyDeselectableItems {
+    var indexPathsForSelectedItems: [IndexPath]? { get }
+    func selectItem(at indexPath: IndexPath?, animated: Bool)
```

Comment:
#pp it's a hard diff to read because of the API change, but this is the new public API, right? I don't love that we lose "smoothly" in the method names. Any reason why that's gone?